rules/falco_rules.yaml

```diff
@@ -3070,7 +3139,10 @@
   condition: (user_known_k8s_client_container)
 
 - rule: The docker client is executed in a container
```

Comment:
This is a legacy rule name and now misleading. We could use this window of opportunity with lots of changes to adjust the rules name? @leogr what are your thoughts?